src/Factory/CouldNotMakeModuleException.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+namespace RebelCode\Modular\Factory;
+
+use Dhii\Modular\Factory\CouldNotMakeModuleExceptionInterface;
+use Dhii\Modular\Factory\ModuleFactoryInterface;
+use Exception;
+
+/**
+ * An exception which occurs if a module factory is unable to create a module.
+ *
+ * @since [*next-version*]
+ */
+class CouldNotMakeModuleException extends ModuleFactoryException implements CouldNotMakeModuleExceptionInterface
```

Comment:
Instead of the trait repos, this class and its ancestor should ideally be in a separate repository.